benchmarks/benchmark/tools/dlio/modules/parallelstore_storage/dataloader_job.tpl

```diff
@@ -0,0 +1,83 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: data-loader-job
+  namespace: ${namespace}
+spec:
+  backoffLimit: 0
+  template:
+    metadata:
+      name: data-loader-job
+      annotations:
+        gke-parallelstore/volumes: "true"
+        gke-parallelstore/cpu-limit: 1000m
+        gke-parallelstore/memory-limit: 5000Mi
```

Comment:
Do we want to support these as var? Just in case the resource not enough, and users can adjust according to their usage?

Reply:
for dataloader I fixed it since it doesn't matter I think.